crypto/rand/drbg_lib.c

```diff
@@ -191,6 +191,9 @@ static void drbg_ossl_ctx_free(void *vdgbl)
 {
     DRBG_GLOBAL *dgbl = vdgbl;
 
+    if (dgbl == NULL)
```

Comment:
This looks like a different fix??

Reply:
Yes, it's thrown in because it got triggered. It's a separate commit. Would you prefer I made it a separate PR?